platform/MCU/newhal-mcu/inc/hw_ticks.h

```diff
@@ -28,10 +28,10 @@ extern "C" {
  * The number of ticks per microsecond of the system counter.
  * SYSTEM_TICK_COUNTER
  */
-#define SYSTEM_US_TICKS		100     // cycles per microsecond
+#define SYSTEM_US_TICKS         (1)     // cycles per microsecond
```

Comment:
Is there any specific reason to change this for `newhal-mcu`?

Reply:
PLATFORM`newhal` is a placeholder for new platforms.  For the most part the numbers associated with it should be some sane default values.  This is why I've changed it to 1, as a default value when a platform doesn't support a hardware based system tick counter.